02-atmos/components/terraform/fetch-location/main.tf

```diff
@@ -0,0 +1,20 @@
+
+# Invoke an our fetch_location.sh external program to get information on the
+# client's IP Address.
+data "external" "fetch_location" {
+  program = ["bash", "${path.root}/fetch_location.sh"]
+}
```

Comment:
I'd prefer replacing with:
```
data "http" "example" {
  url = "https://ipapi.co/json/"

  # Optional request headers
  request_headers = {
    Accept = "application/json"
  }
}
```

https://registry.terraform.io/providers/hashicorp/http/latest/docs/data-sources/http

Reply:
+ a simple `jsondecode`

Reply:
Oh, or was your point here to show that you can call `jq` because it's already installed? maybe i misunderstood the purpose.

Reply:
@osterman yeah that was one of the thoughts. I could definitely see us using the http data source here instead if that is what you'd prefer because both do the trick. I'm likely over complicating things by using `external` since it's a not often used data source. I think I'll update.